modules/graphdb/cmk-parameter-store.tf

```diff
@@ -0,0 +1,130 @@
+resource "aws_kms_key" "graphdb_parameter_store_cmk" {
+  count = var.enable_graphdb_parameter_store_kms_key ? 1 : 0
+
+  description              = var.graphdb_parameter_store_cmk_description
+  customer_master_key_spec = var.graphdb_parameter_store_key_spec
+  is_enabled               = var.graphdb_parameter_store_key_enabled
+  enable_key_rotation      = var.graphdb_parameter_store_key_rotation_enabled
+  tags                     = var.graphdb_parameter_store_key_tags
+  deletion_window_in_days  = var.graphdb_parameter_store_key_deletion_window_in_days
+}
+
+resource "aws_kms_alias" "graphdb_parameter_store_cmk_alias" {
+  count = var.enable_graphdb_parameter_store_kms_key ? 1 : 0
+
+  name          = var.graphdb_parameter_store_cmk_alias
+  target_key_id = aws_kms_key.graphdb_parameter_store_cmk[0].key_id
+}
+
+resource "aws_kms_key_policy" "graphdb_parameter_store_cmk_policy" {
+  count  = var.enable_graphdb_parameter_store_kms_key ? 1 : 0
+  key_id = aws_kms_key.graphdb_parameter_store_cmk[0].id
```

Comment:
Leave an empty line between count and key_id

Reply:
Done.